tests/helm/testdata/goldenfile/logs_otc/options.output.yaml

```diff
@@ -81,7 +81,6 @@ data:
         - flanneld.service
         - format-etcd2-volume.service
         - kube-node-taint-and-uncordon.service
-        - kubelet.service
```

Comment:
how come this is removed?

Reply:
https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/3845/files/d0593424a6a248ee6de7f777b5d321d626aa25d9#diff-deb315cb68ff6989360cadff4ed32ce038f067ba221375952865f7061f9e7930R11

In options.input.yaml, have disabled kubelet .

The case where both systemd and kubelet enabled is checked in [basic.input.yaml](https://github.com/SumoLogic/sumologic-kubernetes-collection/pull/3845/files/d0593424a6a248ee6de7f777b5d321d626aa25d9#diff-f718f99658ab9dd27d0763bc02844f52c81b61147c7e0a44624b1fe303dedb10R8)

@chan-tim-sumo

Reply:
gotcha makes sense!